### PR TITLE
Add split to ATen XLA tensor

### PR DESCRIPTION
**Pull request by asuhan**

### CODE REVIEW

File: torch_xla/csrc/tensor.cpp [hunk 1]
```diff
@@ -941,6 +942,20 @@ XLATensor XLATensor::permute(
                 input.GetDevice());
 }
 
+std::vector<XLATensor> XLATensor::split(const XLATensor& input,
+                                        xla::int64 split_size, xla::int64 dim) {
+  int split_dim = GetCanonicalDimensionIndex(dim, input.shape().get().rank());
```
Comment (dlibenzi) [inline torch_xla/csrc/tensor.cpp]: I'd make input.shape() a local.

File: torch_xla/csrc/data_ops.cpp [hunk 2]
```diff
@@ -214,26 +234,26 @@ xla::XlaOp BuildCat(
   return xla::ConcatInDim(b, cat_inputs, dim);
 }
 
+xla::int64 RoundUpDiv(xla::int64 dividend, xla::int64 divisor) {
```
Comment (dlibenzi) [inline torch_xla/csrc/data_ops.cpp]: third_party/tensorflow/compiler/xla/util.h

CeilOfRatio()

File: torch_xla/csrc/tensor.cpp (hunk 1 above)
Comment (asuhan) [reply torch_xla/csrc/tensor.cpp]: Done.

File: torch_xla/csrc/data_ops.cpp (hunk 2 above)
Comment (asuhan) [reply torch_xla/csrc/data_ops.cpp]: Done.

**Pull request merged**